src/java.base/share/classes/java/lang/reflect/Method.java

```diff
@@ -605,13 +605,16 @@ private Object invoke(Object obj, Object[] args, Class<?> caller)
         return callerSensitive ? ma.invoke(obj, args, caller) : ma.invoke(obj, args);
     }
 
-    @Stable private Boolean callerSensitive;       // lazily initialize
+    //  0 = not initialized (@Stable contract)
+    //  1 = initialized, CS
+    // -1 = initialized, not CS
+    @Stable private byte callerSensitive;
     private boolean isCallerSensitive() {
```

Comment:
A minor nit but a blank line between the declaration and the method would be good here, only to see that the comment is on the field, not the boolean returning method.

Reply:
Added.